guides/common/modules/proc_configuring-for-uefi-http-boot-ipv6-provisioning.adoc

```diff
@@ -0,0 +1,39 @@
+[id="configuring-for-uefi-http-boot-provisioning-in-an-ipv6-network_{context}"]
+= Configuring {Project} for UEFI HTTP Boot Provisioning in an IPv6 Network
+
+Use this procedure to configure {Project} to provision hosts in an IPv6 network with UEFI HTTP Boot provisioning.
+
+.Prerequisites
+
+* Ensure that your clients can access DHCP and TFTP servers.
+
+* Ensure that the UDP ports 67 and 68 are accessible by clients so clients can send DHCP requests and receive DHCP offers.
+
+* Ensure that the TCP port 8000 is open for clients to download files and Kickstart templates from {SmartProxy}.
+
+* Ensure that the host provisioning interface subnet has a TFTP {SmartProxy}, and Templates {SmartProxy} set. For more information, see https://access.redhat.com/documentation/en-us/red_hat_satellite/6.7/html/provisioning_guide/configuring_networking#Configuring_Networking-Adding_a_Subnet_to_the_Satellite_Server[Adding a Subnet to {ProjectServer}] in the _Provisioning Guide_.
+
+.Procedure
+
+. Since clients that are booting from network cannot be identified by a remote IPv6 address due to unmanaged DHCPv6 service, you must disable provisioning tokens. To disable provisioning tokens, navigate to *Administer* > *Settings* > *Provisioning* and set the *Token duration* setting to 0.
```

Comment:
This is a typo made by me - it is actually the very opposite. So ask readers to "ensure the token duration is set to some reasonable time duration and not zero".

Reply:
Oh that is good, @ekohl pointed out that disabling tokens is insecure.
Please elaborate why prov tokens must be enabled here. I do not see the direct correlation between the sentences "{SmartProxy} cannot identify clients that are booting from the network by a remote IPv6 address because of unmanaged DHCPv6 service" and "you must enable provisioning tokens". How are tokens related to Smart Proxy identifying clients?

Reply:
> Oh that is good, @ekohl pointed out that disabling tokens is insecure.

Discussion @ https://github.com/theforeman/foreman-documentation/pull/254#discussion_r482046568.

> This is a typo made by me - it is actually the very opposite. So ask readers to "ensure the token duration is set to some reasonable time duration and not zero".

Glad to hear that tokens should be used.

The default for `token_duration` is is 6 * 60 minutes, so 6 hours. It's probably good to recommend users who are unsure about the exact value to choose the default, right @lzap?

Reply:
Yeah we actually do not require any action here to be honest. I wonder if we should delete this line from the procedure. We can perhaps move this to Requirements only in the form of "Ensure tokens are not disabled...".

I am just doublechecking, I wonder why I added it initially. For IPv6 DHCP is unmanaged, therefore system boots up, picks (random) IPv6, gets TFTP config which contains the token, yeah. This will work, no reason to disable them.